src/backend/src/controllers/teams.controllers.ts

```diff
@@ -23,4 +24,19 @@ export default class TeamsController {
       next(error);
     }
   }
+
+  static async setMembers(req: Request, res: Response, next: NextFunction) {
+    try {
+      const { userIds } = req.body;
+      const submitter = await getCurrentUser(res);
+
+      // update the team with the input fields
+      const updateTeam = await TeamsService.updateSingleTeam(submitter, req.params.teamId, userIds);
```

Comment:
the service function name should be the same as the controller name (both should be setMembers)

Reply:
actually let's go with `setTeamMembers` for both. that's what the api and hook uses and i think that's the most clear